js/kucoin.js

```diff
@@ -302,6 +302,8 @@ module.exports = class kucoin extends Exchange {
             price = this.safeFloat (order, 'dealPrice');
         if (typeof price === 'undefined')
             price = this.safeFloat (order, 'dealPriceAverage');
+        if (price === 0.0)
```

Comment:
Type checking doesn't work with Python like that, unfortunately.

Reply:
Ok. I suppose you will do that? I didn't understand what you said so :)